packages/core/tests/acceptance/line-chart-test.ts

```diff
@@ -124,8 +121,7 @@ module('Acceptance | line chart', function(hooks) {
     await visit('/line-chart');
 
     // switch on `stacked` and expand the config
-    await click('.chart-container.dimension .line-chart-config__stacked-opt .denali-switch');
-    await click('.chart-container.dimension .line-chart-config__series-config__header');
+    await click('.chart-container.dimension .line-chart-config__stacked-opt-select');
```

Comment:
```suggestion
    await click('.line-chart-config__stacked-opt-select');
```

Reply:
we need these. Its on a page with many line charts so the .dimension differentiates between them

Reply:
ah, makes sense 👍